README.md

```diff
@@ -89,12 +89,13 @@ This module will contain all tests being written against a SAML IdP. The `src` d
 
 This [coverage](ctk/idp/coverage) directory is used to track which sections of each SAML specification are covered by these tests:
 
-#### sp
-This module will contain all tests being written against a SAML SP. The src directory of the module is organized identically to the idp module.
-
```

Comment:
It feels so good to have all this orphaned SP stuff removed. So much cleaner!

Reply:
☝️